### DISCUSSION TITLE
Testing via connected device/debug or Play Store download

### DISCUSSION THREAD

**Discussion by edgfowler** [KyleKun/one_second_diary#7] [General]
Kyle and/or Testers,
Do you have a preference and what feedback, information, logs, etc. is most helpful?

**Comment by KyleKun** [comment]
Hey, 

You can test the Play Store version, it's the same as the apk released here.
No need for logs, just try to provide as much details as possible if it is an issue or explain with examples if it's a feature proposal.
I'm not creating an Issue Template for now since it's a small project.

Thanks!
